client/handler.go

```diff
@@ -21,7 +21,9 @@
 package client
```

Comment:
Could you still add a test for this handler please? :)

Reply:
@aeneasr done :)